scripts/release_scripts/initial_release_prep.py

```diff
@@ -192,24 +170,13 @@ def main():
         release_constants.ONE_TIME_JOBS_SPREADSHEET_URL)
     common.ask_user_to_confirm(
         'Please copy the names of the jobs to be run for this release along '
-        'with author names, author mail ids & instruction docs.')
```

Comment:
Are we expecting to change line 142 with the new arg?

Reply:
Yes, updated.